### PR TITLE
Update alert.md

### PR DESCRIPTION
**Pull request by GnatalieH**
Began adding Placement guidance for Default and Background only alerts.

### CODE REVIEW

**Review by humancompanion-usds — CHANGES_REQUESTED** [review]
I'm not sure the examples relate to the changes we're making here to better define background-only Alerts. Also, let's try to minimize the example images to just showing what is relevant rather than the the entire page.

File: src/_components/alert.md [hunk 1]
```diff
@@ -120,6 +120,22 @@ When the user is required to do something in response to an alert, let them know
...
+#### Default alerts
+
+* In most cases, the Default alert (in all of its variations) should be placed directly below the H1 and intro text, near the top of the page. 
+* When a Default alert is applicable to a specific section of content on a page, it should be placed directly below the header of that section.
+
+EXAMPLES TBA
+
+#### Background color only alerts
+
+* Background color only alerts should be placed directly below the H1 and intro text when the alert applies to the entire page.
```
[the remark this answers is not in the document]
Comment (GnatalieH) [reply src/_components/alert.md:134]: > What is the use case for background-only Alerts that apply to the entire page?

image (19)

**Comment by humancompanion-usds** [comment]
### Background only placement and rules

* Appropriate when placed within a container such as a Card or other box.
* Errors related to a field or section are below the label, legend, or header.
* Info can appear between a section.
*

**Comment by GnatalieH** [comment]
@humancompanion-usds I changed Previous to Back and am hoping for your review so we can merge this PR. Thanks!

**Review by humancompanion-usds — APPROVED** [review]

**Pull request merged**